data.sql

```diff
@@ -10,3 +10,26 @@ VALUES ('Pikachu', '2021-01-07', 1,false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', '2017-05-12', 5,true, 11.00);
 
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020-02-08', 0, false, -11.00);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', '2021-11-15', 2, true, -5.7);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Squirtle', '1993-04-02', 3, false, -12.13);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Angemon', '2005-06-12', 1, true, -45);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Boarmon', '2005-06-07', 7, true, -20.4);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Blossom', '1998-10-13', 3, true, -17);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Ditto', '2022-05-14', 4, true, -22);
```

Comment:
- [ ] The weight_kg for these two animals does not have the `-` sign in front of them. Kindly remove the negative(-) sign in front of the numbers. It should be 17 and 22 respectively. Changing them will save you in the long run with queries relating to the kgs. 👍

Reply:
The issue has been fixed. Thanks for pointing that out.